### PR TITLE
Travel insurance for travel fellowships

### PR DESCRIPTION
**Pull request by peterjc**
This attempts to capture what was discussed by the OBF board today.

### CODE REVIEW

**Review by gedankenstuecke — APPROVED** [review]
LGTM!

**Review by cjfields — APPROVED** [review]
Looks great!

File: Travel_fellowships.md [hunk 1]
```diff
@@ -92,6 +92,12 @@ However, this fellowship may cover the rental cost of the large hardware (for ex
 The cost will be capped to a maximum of $40 (USD) each for small hardware such as headphones, webcam, speaker, and/or microphone.
 If the cap is insufficient, applicants can describe their hardware needs in their application.
 
+### Expenses and Cancellation
+
+We recommend you purchase travel insurance, and include this as part of your expenses claim.
+In the event of cancellation, we would expect to reimburse your travel insurance premium and any insurance excess fee.
```
Comment (HLWiencko) [inline Travel_fellowships.md:98]: The premium is what the person pays regardless, right? 

The excess can be much higher than the premium, and might be over the $1000 cap. We may want to say something like "as long as the total reimbursement does not exceed the $1000 award."

**Review by HLWiencko — APPROVED** [review]
Approved, with a couple comments.

File: Travel_fellowships.md (hunk 1 above)
Comment (peterjc) [reply Travel_fellowships.md:98]: True - although I don't recall seeing an excess that high personally.

**Review by HLWiencko — APPROVED** [review]

File: Travel_fellowships.md [hunk 2]
```diff
@@ -92,6 +92,13 @@ However, this fellowship may cover the rental cost of the large hardware (for ex
 The cost will be capped to a maximum of $40 (USD) each for small hardware such as headphones, webcam, speaker, and/or microphone.
 If the cap is insufficient, applicants can describe their hardware needs in their application.
 
+### Expenses and Cancellation
+
+We recommend you purchase travel insurance, and include this as part of your expenses claim.
+In the event of cancellation, we would expect to reimburse your travel insurance premium and any insurance excess fee (as long as the total reimbursement does not exceed the originally agreed award).
+
+If the awardee elects not to purchase travel insurance, any reimbursement is at the discretion of the board.
```
Comment (nlharris) [inline Travel_fellowships.md:100]:
```suggestion
If the awardee elects not to purchase travel insurance, any reimbursement is at the discretion of the Board.
```

Comment (nlharris) [inline Travel_fellowships.md:100]: Just noting that this document switches back and forth between 2nd person ("you"), third person singular ("the awardee"), and third person plural ("applicants"). Probably not worth fixing, though?

**Review by nlharris — APPROVED** [review]

**Review by cjfields — APPROVED** [review]

File: Travel_fellowships.md (hunk 2 above)
Comment (cjfields) [reply Travel_fellowships.md:100]: > Just noting that this document switches back and forth between 2nd person ("you"), third person singular ("the awardee"), and third person plural ("applicants"). Probably not worth fixing, though?

Yeah I think that's worth maybe revising, maybe in a followup pull PR?

**Review by gedankenstuecke — APPROVED** [review]
Just leaving an updated approval after the minor changes 👍

**Pull request merged**